src/include/processor/execution_context.h

```diff
@@ -15,12 +15,15 @@ struct ExecutionContext {
     storage::BufferManager* bufferManager;
     main::ClientContext* clientContext;
     common::VirtualFileSystem* vfs;
+    main::Database* database;
```

Comment:
After this PR, let's try to clear `mm, bm & vfs` and replace them with just `clientContext or executionContext` ideally